src/components/LeftDrawer/LeftDrawer.module.css

```diff
@@ -66,14 +66,14 @@
   width: 100%;
   height: 52px;
   border-radius: 8px;
-  background-color: var(--bs-white);
   display: flex;
   align-items: center;
+  padding: 2.5rem 1rem 2.5rem 1rem;
 }
 
 .leftDrawer .profileContainer:focus {
   outline: none;
-  background-color: var(--bs-gray-100);
+  background-color: #5ac989;
```

Comment:
Don't hard code the colours, use css variables

Reply:
Sure, just noticed that the color is being taken care of by button variant so I will remove it